app/_data/docs_nav_konnect.yml

```diff
@@ -31,6 +31,14 @@
           url: /getting-started/spec/implement
```

Comment:
I think implement should come before publish? WDYT?
Import > Implement > Publish > Register.

Reply:
Yep, that makes sense. Less jumping back and forth.